contracts/MerkleProof.sol

```diff
@@ -1,3 +1,4 @@
+pragma solidity ^0.4.4;
```

Comment:
File can be deleted.

Reply:
Fixed with https://github.com/TrueBitFoundation/scrypt-interactive/pull/24/commits/11f1b916dd3679e133be7d9fa59e4541329d071d